views/index.pug

```diff
@@ -19,35 +19,35 @@ html
       | : Returns a list of all Pokémon Go Battle leagues.
     h1
       span.get GET
-      | /leagues/:league/leads
+      | /leagues/:leagueSlug/leads
     code
       span.desc Method
       | : 
       span.get GET<br>
       span.desc Route
-      | : https://placeholder.url/leagues/:league/leads<br>
+      | : https://placeholder.url/leagues/:leagueSlug/leads<br>
       span.desc Description
       | : Returns a list of all lead Pokémon players have encountered in the specified league.
     h1
       span.get GET
-      | /leagues/:league/leads/{slug}
+      | /leagues/:leagueSlug/leads/{slug}
```

Comment:
I had to guess the lead slug since the list route only returns names and not slugs. Also this URL doesn't match the example below.

![image](https://user-images.githubusercontent.com/5775716/83802060-1d889c80-a678-11ea-9435-ba06745cfa20.png)